res/TensorFlowLiteRecipes/Net_Preactivation_BN_000/test.recipe

```diff
@@ -0,0 +1,176 @@
+operand {
+  name: "Hole"
+  type: FLOAT32
+  shape {
+    dim: 64
+    dim: 12
+    dim: 12
+    dim: 64
```

Comment:
if possible, can you use a small number for testing? current numbers can slow the test and CI

Reply:
I've reduced the dimension.

Model size decreased from 2.4 MB to 19.9 kB.